src/System.CommandLine.Benchmarks/CommandLine/Perf_Parser_Directives_Suggest.cs

```diff
@@ -22,10 +22,16 @@ public void Setup()
         {
             _nullConsole = new NullConsole();
 
+            Option<string> fruitOption = new("--fruit");
+            fruitOption.CompletionSources.Add("apple", "banana", "cherry");
```

Comment:
This naming now looks wrong. The things being added here are really completion items, not completion sources.

Reply:
@jonsequitur I've tried renaming the `Add` extension method to `AddCompletionItems` but it break the C# duck typing for collection initialization:

![image](https://user-images.githubusercontent.com/6011991/212275701-1f43112c-16fd-4c97-bd01-9074957f5ad0.png)

```log
Error	CS1503	Argument 1: cannot convert from 'string' to 'System.Func<System.CommandLine.Completions.CompletionContext, System.Collections.Generic.IEnumerable<System.CommandLine.Completions.CompletionItem>>'	System.CommandLine.Tests (net462), System.CommandLine.Tests (net7.0)	D:\projects\command-line-api\src\System.CommandLine.Tests\CompletionTests.cs	125	Active
```